Keyboards/KeyboardsBase/KeyboardViewController.swift

```diff
@@ -1207,6 +1207,13 @@ class KeyboardViewController: UIInputViewController {
 
   /// Sets up all buttons and labels for the conjugation view given constraints to determine the dimensions.
   func setConjugationBtns() {
+    // add swipe functionality to the conjugation and declension views
```

Comment:
Quick note here that we want comments on their own line to be complete sentences with a period at the end :) No stress, I'll fix it 😊